### PR TITLE
added optional default to border box display

### PR DESCRIPTION
**Pull request by anEffingChamp**
I typically prefer that my content default to the border box model, and optionally be set to the content, or padding box models. To that end I wrote this reset to size all elements that way, and group it with the grid component.

### CODE REVIEW

**Comment by jamesalley** [comment]
I would be cautious about adding `* { box-sizing: border-box;}` to base. I want to love it. I used this rule in my last big project with great success, in that I found dimensions easier to grok and work with. But there were drawbacks, some minor and some serious, which lead me to temper my love for border-box.

The first drawback is that browsers still have bugs in their implementations of border-box. I've noticed rendering quirks from time to time.

The second drawback relates to interoperability. If we start with  `* { box-sizing: border-box;}`, this will cause conflict when adding in modules which are expecting content-box. All such modules would have to have their box-model reset.

Put simply, while @ericf (or even I) might add this to every web project he does, not everyone will want it, and we should not force them to override  `* { box-sizing: border-box;}`.

**Comment by jamesalley** [comment]
Even though this would be optional (not included in the rollup) I don't think we should include it. One, no one will want to make another request just for this little file. Two, it's trivial to implement yourself in your app.css.
